tests/generic/check_base_trades_seed.sql

```diff
@@ -0,0 +1,10 @@
+-- this tests checks a dex base_trades model for every row in a seed file.
+-- actual implementation in macros/test-helpers/check_seed.sql
+{% test check_base_trades_seed(model, seed_file, filter=None) %}
```

Comment:
i like the idea here

can we be more specific on naming?
`check_dex_base_trades_seed` (add `dex` in there to help differentiate future sectors that follow)

Reply:
I just renamed the test to `check_dex_base_trades_seed`